app/javascript/.storybook/config.js

```diff
@@ -1,5 +1,5 @@
-import { configure } from '@storybook/react';
-import '../../assets/stylesheets/minimal.scss';
+import { configure } from '@storybook/preact';
+// import '../../assets/stylesheets/minimal.scss';
```

Comment:
I'm commenting this out for now. Storybook could not load our `minimal.scss` because we are using Rail's `image_path` helper in it. This can be fixed by removing those css background images and use image tag in the appropriate area.